cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
> For tiny benefit I wonder why we need such goroutine/channel code

It is not a tiny benifit @balamurugana it affects the UI on the console. Refer to my comments earlier in the PR on why this go-routine was added.

Basically we in this commit previously we were printing:
```
Initializing FS backend format.. Waiting for lock to be acquired.
Lock acquired, safely proceeding to intialize object layer.
```

everytime minio was started. This message is only useful in the rare case of another process migrating the backend and your lock will block. Hence it is worth a little complexity in the code to print it only when it is needed and not print all the time.